src/Main.java

```diff
@@ -1,5 +1,69 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello World!");
+
+        System.out.println("Задача 1");
+        var dog = 8.0;
+        System.out.println(dog);
+        var cat = 3.6;
+        System.out.println(cat);
+        var paper = 763789;
+        System.out.println(paper);
+
+        System.out.println("Задача 2");
+        dog = dog+4;
+        cat = cat+4;
+        paper = paper+4;
+        System.out.println(dog);
+        System.out.println(cat);
+        System.out.println(paper);
+
+        System.out.println("Задача 3");
+        dog = dog-3.5;
+        cat = cat-1.6;
+        paper = paper-7639;
+        System.out.println(dog);
+        System.out.println(cat);
+        System.out.println(paper);
+
+        System.out.println("Задача 4");
+        var friend = 19;
+        System.out.println(friend);
+        friend = friend+2;
+        System.out.println(friend);
+        friend = friend/7;
+        System.out.println(friend);
+
+        System.out.println("Задача 5");
+        var frog = 3.5;
+        System.out.println(frog);
+        frog = frog*10;
+        System.out.println(frog);
+        frog = frog/3.5;
+        System.out.println(frog);
+        frog = frog+4;
+        System.out.println(frog);
+
+        System.out.println("Задача 6");
+        var boxer1 = 78.2;
+        var boxer2 = 82.7;
+        var result1 = boxer1+boxer2;
+        System.out.println(result1);
+        var result2 = boxer1-boxer2;
+        System.out.println(result2);
+
+        System.out.println("Задача 7");
+        var result3 = boxer2%boxer1;
+        System.out.println(result3);
+
+        System.out.println("Задача 8.1");
+        var totalHours = 640;
+        var hoursPerEmployee = 8;
+        var numberOfWorkers = totalHours/hoursPerEmployee;
+        System.out.println("Всего работников в компании — " + numberOfWorkers + " человек.");
+
+        System.out.println("Задача 8.2");
+        var increasedWorkers = numberOfWorkers+94;
```

Comment:
```suggestion
        numberOfWorkers = numberOfWorkers+94;
```
можно было не создавать новую переменную, а просто поменять текущую